docs/installation.rst

```diff
@@ -40,8 +41,8 @@ Installing the latest stable version is possible either using pip or conda.
 Using pip
 ---------
 
-To install **healpix** with `pip <http://www.pip-installer.org/en/latest/>`__
-from `PyPI <https://pypi.python.org/pypi/regions>`__
+To install ``healpix`` with `pip <http://www.pip-installer.org/en/latest/>`__
```

Comment:
In documentation I don't like using code notation to refer to the actual package (when not in the context of imports), hence why I use bold to distinguish it. This will be more apparent if we rename to ``astropy-healpix`` which looks like code but definitely isn't the import. What do you think?